app/code/Magento/Wishlist/Block/Customer/Wishlist.php

```diff
@@ -63,6 +68,7 @@ public function __construct(
         $this->_helperPool = $helperPool;
         $this->currentCustomer = $currentCustomer;
         $this->postDataHelper = $postDataHelper;
+        $this->getWishlistItems();
```

Comment:
Please remove this unnecessary method call

Reply:
this is not unnecessary method call, without calling this method collection will not get load

Reply:
The collection will be loaded once requested, the method call is unnecessary because you do not need the collection here